website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
@lemoness Thoughts on showing the notes popover on click rather than hover?

I'm anticipating numerous bug reports from people saying their notes won't appear if we make that switch.